spec/rails_helper.rb

```diff
@@ -1,4 +1,4 @@
-ENV['RAILS_ENV'] ||= 'test'
+ENV['RAILS_ENV'] = 'test'
```

Comment:
I don't imagine there is a time when we'd run the tests outside of the 'test' environment, but this would probably make for inscrutable errors if we had a separate environment for tests on Travis vs. local machine. Is there a reason you changed this here (not saying it's wrong, just curious)